uwblueprint/paramedics-react
# Move to offix client

[Notion ticket](https://www.notion.so/uwblueprintexecs/Replace-Apollo-Client-with-Offix-7728294826a64d6c8c453c3c2c178288)

## Changes
- Replace existing apollo client with offix client

## Testing
- Verify the following queries & mutations still work in the FE
Events
http://localhost:3000/events
- View all events
- Add
- Edit
- Archive
- Unarchive
- Delete

Team Members / Users
http://localhost:3000/manage/members
- View
- Add
- Edit
- Delete

Ambulances
http://localhost:3000/manage/ambulances
- View
- Add
- Edit
- Delete

Hospitals
http://localhost:3000/manage/hospitals
- View
- Add
- Edit
- Delete

Collection Points
http://localhost:3000/events/{eventId}
- View
- Add
- Edit
- Delete

Patients
http://localhost:3000/events/{eventId}/ccps/{ccpId}
- View
- Add
- Edit
- Delete

Location Pins
http://localhost:3000/events/{eventId}/map
- Add

## Checklist
>[Code review doc for reference](https://www.notion.so/uwblueprintexecs/Code-Review-a21fc85b00394f488e92d9d605f6b2bc)

before opening PR

filling out PR

after opening PR
